### ISSUE TITLE
Replace jargon 'map' with 'entity object' on header/footer docs page

### ISSUE THREAD

**Issue by teleksterling** [home-assistant/home-assistant.github.io#32366]
### Feedback

Explanatory text under Buttons header uses technical language (map) when explaining the alternative to using an entity id.

I believe elsewhere the 'entity data structure' is referred to as an "entity object". This term is more generally used and understood beyond a programming audience. 

Example of suggested usage: 
https://www.home-assistant.io/dashboards/history-graph#entities

>   entities list REQUIRED
>   A list of entity IDs or entity objects, see below.

### URL

https://www.home-assistant.io/dashboards/header-footer/

### Version

2024.4.3

**Comment by frenck** [comment]
Except, In the frontend, it ain't the entity object 🤷

**Comment by teleksterling** [comment]
True. But the entire page (and many others) are written with no mention of configuration via the frontend/UI.

(Honest question) Is the intention to only make doc revisions if it's to remove/ augment yaml configuration guidance with UI-based setup?
